datafusion/src/physical_plan/planner.rs

```diff
@@ -210,6 +243,24 @@ impl PhysicalPlanner for DefaultPhysicalPlanner {
         let plan = self.create_initial_plan(logical_plan, ctx_state)?;
         self.optimize_plan(plan, ctx_state)
     }
+
+    /// Create a physical expression from a logical expression
+    /// suitable for evaluation
+    ///
+    /// `e`: the expression to convert
+    ///
+    /// `input_dfschema`: the logical plan schema for evaluating `e`
+    ///
+    /// `input_schema`: the physical schema for evaluating `e`
+    fn create_physical_expr(
+        &self,
+        e: &Expr,
```

Comment:
Nit: why call it `e`? I think `expr` as variable name is more consistent with the whole codebase?

Reply:
I agree `expr` is more consistent -- I think I was following the model of `DefaultPhysicalPlanner::create_physical_expr`. I will change